client/state/selectors/test/get-jetpack-remote-install-error.js

```diff
@@ -0,0 +1,29 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { getJetpackRemoteInstallError } from 'state/selectors';
+
+const url = 'https://yourgroovydomain.com';
+
+describe( 'getJetpackRemoteInstallError()', () => {
+	test( 'should return null if no errors', () => {
+		const state = {
+			jetpackRemoteInstall: {
+				error: {},
+			},
+		};
+		expect( getJetpackRemoteInstallError( state, url ) ).toBe( null );
+	} );
+	test( 'should return any existing error', () => {
+		const state = {
+			jetpackRemoteInstall: {
+				error: {
+					[ url ]: 'SOME_ERROR',
+				},
+			},
+		};
+		expect( getJetpackRemoteInstallError( state, url ) ).toEqual( 'SOME_ERROR' );
```

Comment:
[`toEqual()`](https://facebook.github.io/jest/docs/en/expect.html#toequalvalue) is really for object equalits, prefer [`toBe( 'SOME_ERROR' )`](https://facebook.github.io/jest/docs/en/expect.html#tobevalue).